src/main/java/lotto/model/RandomNumberGenerator.java

```diff
@@ -0,0 +1,20 @@
+package lotto.model;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class RandomNumberGenerator implements NumberGenerator {
+
+    private static final int MIN_NUMBER = 1;
+    private static final int MAX_NUMBER = 45;
+    private static final int SIZE = 6;
+    private static final List<Number> numberList
+            = IntStream.rangeClosed(MIN_NUMBER, MAX_NUMBER).mapToObj(Number::new).collect(Collectors.toList());
+
+    public List<Number> pickNumbers() {
+        Collections.shuffle(numberList);
+        return Collections.unmodifiableList(numberList.stream().limit(SIZE).collect(Collectors.toList()));
```

Comment:
```suggestion
    private static final List<Number> numberList = IntStream.rangeClosed(MIN_NUMBER,MAX_NUMBER)
            .mapToObj(Number::new)
            .collect(Collectors.toList());

    public List<Number> pickNumbers() {
        Collections.shuffle(numberList);
        return Collections.unmodifiableList(numberList.stream()
                .limit(SIZE)
                .collect(Collectors.toList()));
```
개인적으로는 메서드 체이닝시 개행을 해주는게 조금 더 가독성이 좋긴하더라구요
최종연산 전까지는 파이프라인을 통해 단계별로 실행되는 부분때문에 한줄보다는 더 개행이 의미가 있다고 생각하기도 하는데요 
민성님 생각도 들려주시면 감사하겠습니다~

Reply:
아직 스트림 사용이 어색해서 자동 정렬만 적용했는데, 
말씀 해주신대로 개행을 사용하는게 가독성도 좋고 의미 분리도 되어서 좋은것 같습니다!